src/histogram-inl.h

```diff
@@ -5,10 +5,80 @@
 
 #include "histogram.h"
 #include "node_internals.h"
+#include "env-inl.h"
+#include "v8.h"
 
 namespace node {
 
-inline Histogram::Histogram(int64_t lowest, int64_t highest, int figures) {
+using v8::FunctionCallbackInfo;
+using v8::Local;
+using v8::Map;
+using v8::Number;
+using v8::Value;
+
+inline void Histogram::GetMin(
+    Histogram* histogram,
+    const FunctionCallbackInfo<Value>& args) {
+  double value = static_cast<double>(histogram->Min());
```

Comment:
`Min()` returns int_max which results in `min: 9223372036854776000` if no measurement was done (e.g. after `reset()`) which looks not that nice. Maybe set `min/max` to `NAN` in case `count` is 0?